src/main/resources/data/products.json

```diff
@@ -1,44 +1,51 @@
 [
   {
     "id": "1",
-    "productKindId": "1",
+    "productKindId": "kind1",
     "productKindName": "Current Account",
     "productTypeId": "1111111",
     "productTypeName": "Current Account"
   },
   {
     "id": "2",
-    "productKindId": "2",
+    "productKindId": "kind2",
     "productKindName": "Savings Account",
     "productTypeId": "2222222",
     "productTypeName": "Savings Account"
   },
   {
     "id": "3",
-    "productKindId": "3",
+    "productKindId": "kind3",
+    "productKindName": "Debit Card",
+    "productTypeId": "333333",
+    "productTypeName": "Debit Card"
+  },
+  {
+    "id": "4",
```

Comment:
threre is a correlation with the productIds in ``seed/product-group-seed.json``. If you change increment ids here you should do it there as well .

also to make it more visible the ``product-group-seed.json`` relates with these ids I would suggest to move the file one hierarchy lower from the seed folder.

Reply:
1. Check id correlation in product-group-seed.json - done.

2. Regarding the moving - do you suggest to move product-group-seed.json out of seed/ ?